compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -836,7 +835,7 @@ object SpaceEngine {
     def isCheckable(tp: Type): Boolean =
       val tpw = tp.widen.dealias
       val classSym = tpw.classSymbol
-      classSym.is(Sealed) ||
+      classSym.is(Sealed) && !tpw.isLargeGenericTuple || // exclude large generic tuples from exhaustivity
```

Comment:
Could the comment highlight that this is a limitation and what would need to be done to lift this limitation?